influxclient/model/templates/client.tmpl

```diff
@@ -0,0 +1,58 @@
+
+// Doer performs HTTP requests.
+//
+// The standard http.Client implements this interface.
+type HTTPRequestDoer interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+// Client which conforms to the OpenAPI3 specification for this service.
+type Client struct {
```

Comment:
Shouldn't we rather rename this `Client` to `GeneratedClient` to disabiguate `influxclient/model/Client` and `influxclient/Client`